src/select/SimpleSelect.js

```diff
@@ -649,7 +629,7 @@ class SimpleSelect extends Component {
     const hasStaticOptions =
       (Array.isArray(staticOption) && !!staticOption.length) || !!staticOption
     if (opts.length || hasStaticOptions) {
-      const MenuComponent = this.props.menuComponent
+      const MenuComponent = menuComponent
       menu = (
         <div className="crane-select-menu-container" role="listbox">
```

Comment:
I think to fix the a11y issue we want to remove this role and add `id='crane-select-menu-container`

Reply:
Also to test I think you'll have to use a select component that allows filtering and the menu has to be open when you run the a11y tests.

Reply:
@sthomas1618 did this one get fixed?